### PR TITLE
Updated the readme simplist example.

### PR DESCRIPTION
**Pull request by lordadamson**
made it a bit more complicated, but at least anyone reading won't have to ask amiright lol

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,12 +43,41 @@ the application uses.
...
+└── usr
+    ├── bin
+    │   └── your_app
+    ├── lib
+    └── share
+        ├── applications
+        │   └── your_app.desktop
+        └── icons
+            └── hicolor
+                └── 256x256
```
Comment (TheAssassin) [inline README.md]: Replace this with some kind of template. I suggest:

```
└── <theme>
    (for example, hicolor, ...; see https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html for details)
    └── <resolution>
        (for example, 256x256, scalable, ...; see https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html for details)
```

File: README.md [hunk 2]
```diff
@@ -43,12 +43,41 @@ the application uses.
...
+
+* Notice that both `Exec` and `Icon` only have file names.
+* Also Notice that the `Icon` entry does not include an extension.
 
-`linuxdeployqt path/to/appdir/usr/share/application_name.desktop`
+Read more about desktop files in the [freedesktop specification here](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html).
 
-Where the _desktop_ file specifies the executable to be run (with `EXEC=`), the name of the applications and an icon.
-See [desktop file specification](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html).
+Now you can say: `linuxdeployqt path/to/AppDir/usr/share/applications/your_app.desktop`
```
Comment (TheAssassin) [inline README.md]: `s/linuxdeployqt/linuxdeployqt-continuous-x86_64.AppImage/g`

**Comment by TheAssassin** [comment]
Thanks. Please see my comments.

File: README.md (hunk 2 above)
Comment (lordadamson) [reply README.md]: lol okay :smile:

**Comment by lordadamson** [comment]
Alright, I updated the PR.

**Comment by probonopd** [comment]
Thank you very much @lordadamson - this kind of contribution is __highly__ appreciated.

**Comment by lordadamson** [comment]
You're very welcome :smile:

**Pull request merged**